diff --git a/node_modules/@types/geojson/index.d.ts b/node_modules/@types/geojson/index.d.ts
index b64392c..78bd17b 100644
--- a/node_modules/@types/geojson/index.d.ts
+++ b/node_modules/@types/geojson/index.d.ts
@@ -28,7 +28,7 @@ export type BBox = [number, number, number, number] | [number, number, number, n
  * The previous GeoJSON specification allowed more elements (e.g., which could be used to represent M values),
  * but the current specification only allows X, Y, and (optionally) Z to be defined.
  */
-export type Position = number[]; // [number, number] | [number, number, number];
+export type Position = [number, number];
 
 /**
  * The base GeoJSON object.
